vanhouse/src/App.js

```diff
@@ -1,35 +1,109 @@
-import React from 'react'
+import React,{useState}from 'react'
 import {BrowserRouter as Router, Route} from "react-router-dom";
-import {Container,Row,Col} from 'react-bootstrap';
+import {InputGroup, Button, FormControl, DropdownButton,ButtonToolbar,Container,Col,Row} from 'react-bootstrap';
+
+import {MenuItem,Dropdown} from 'bootstrap';
 import PostDetail from "./components/PostDetail";
 import Header from "./components/Header";
 import UserList from './components/UserList';
+import Price from './components/Price';
+import Location from './components/Location';
+import Category from './components/Category';
 import "./App.css";
 import PostCollection from "./components/PostCollection";
+import search from './assets/search.png';
 
 function App() {
+  const [leftState,setLeftState] = useState(0);
+  const [left,setLeft] = useState(<UserList />);
+
+  const style = {
+    display: 'flex',
+    margin:'auto',
+    textAlign: 'center',
+    width:'70%'
+
+
+}
```

Comment:
To keep style consistent, I think it's better to put css into a separate file. Same to other styles you made in component files.